### ISSUE TITLE
All non-docs links to observablehq.com are broken unless they open in new tab

### ISSUE THREAD

**Issue by tophtucker** [observablehq/documentation#136]
Followup to https://github.com/observablehq/documentation/issues/134; @eagereyes fixed in the case of the pricing badge, but it looks like the issue applies to any link to the main observablehq.com site. E.g. the link to the module require debugger is broken on https://observablehq.com/documentation/debugging/require-stubborn-modules; it shows the correct URL but 404s. Something funky’s going on with pushState and routing.

**Comment by eagereyes** [comment]
Good catch! I think the solution for now is to replace all links with markdown links. I'll create a PR, should be pretty easy.

Ideally of course, this would be fixed in VitePress, I'll see if they already have an issue open.

I've seen some differences between the `docs:dev` and `docs:build` versions of VitePress, which probably explains why things are sometimes different between local and prod.

**Comment by mbostock** [comment]
The problem is that VitePress thinks it’s an internal link rather than an external one. I think the fix is that you have to use the full URL in the link (including https://observablehq.com). You can’t just use a path when navigating out of the docs.

**Comment by eagereyes** [comment]
They are complete links! That's what so confusing about this. How does VitePress even know this is the same site? We don't have the domain in the configuration, so this must happen at runtime.

**Comment by tophtucker** [comment]
One fairly prominent spot where this is still broken is “Next page: Tutorials” on https://observablehq.com/documentation/learn/overview. (I get why we want Tutorials in the side nav but it does seem odd that “Next page” on the first page takes you out of the flow of documentation pages.)

<img width="1420" alt="image" src="https://github.com/observablehq/documentation/assets/841829/e633bec6-023e-4a7c-a96d-b28cb47e8d61">

**Comment by eagereyes** [comment]
I don't actually think that these previous/next links actually make a ton of sense on the documentation site. They're driven by the sidebar, which is a mix of links to external pages, docs pages, and anchors within pages, which causes weird behavior in a few places (including this one). Even if you'd want to click through the pages, you'd end up in loops when you get to a page where we have sidebar links within the page.

So long story short, I think we should remove them from this site.

**Comment by eagereyes** [comment]
I just [created a PR](https://github.com/observablehq/documentation/pull/146) that manually sets the prev/next links for the pages where it's necessary to skip over external links (and also the ones with internal links to avoid the infinite loop issue)
